openadapt/app/cards.py

```diff
@@ -37,6 +37,29 @@ async def pick_file():
     import_dialog.open()
 
 
+def stop_record(tray):
+    global PROC
+    if PROC is not None:
+        tray.notify("OpenAdapt", "Stopping recording...")
+        PROC.send_signal(signal.SIGINT)
+
+        # wait for process to terminate
+        PROC.wait()
+        tray.remove_notification()
+        tray.notify("OpenAdapt", "Recording finished.")
+        PROC = None
+    else:
+        tray.notify("OpenAdapt", "No recording in progress.")
+
+
+def quick_record():
+    global PROC
```

Comment:
From https://peps.python.org/pep-0008/#constants:

> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

PROC does not seem to be a constant since we are assigning it here. What do you think about naming this `record_proc` or similar?

Reply:
sure